sphinx_autobuild/build.py

```diff
@@ -31,13 +28,11 @@ def __call__(self, *, rebuild: bool = True):
             )
             raise
 
-        show(command=["sphinx-build"] + self.sphinx_args)
-
-        # NOTE:
-        # sphinx.cmd.build.build_main is not considered to be public API,
-        # but as this is a first-party project, we can cheat a little bit.
-        if return_code := build_main(self.sphinx_args):
-            print(f"Sphinx exited with exit code: {return_code}")
+        try:
+            show(command=["sphinx-build"] + self.sphinx_args)
```

Comment:
This doesn’t need to be inside the try block.

Reply:
@AA-Turner fixed it, see [diff](https://github.com/sphinx-doc/sphinx-autobuild/compare/f440a7415d7f5e58bac22ec9b549abdddff2e210..050a5f6bb8a96fb1807bb0efee2317c4f2668e75)